.github/workflows/ruby.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
Some newline inconsistencies in this file. Please be consistent about blank line use, and add a newline at the end of the file.

Reply:
Still some newline/indentation inconsistencies in this file.